### ISSUE TITLE
broken Lighting \ Rendering in both eyes

### ISSUE THREAD

**Issue by Fan4eGGG** [Nibre/MotherVR#253]
Salute!
I've seen posts already here such as #248 and #154 and I'm still trying to figure out if all this advice has helped anyone, because it hasn't. I've tried reducing FOV from max. to min, rendering quality from 2.0 to 0.5, put mod [aliasIsolation](https://github.com/aliasIsolation/aliasIsolation), uninstalled and reinstalled the game, tried to run in **SteamVR** and in **OculusVR**, changed graphics settings, changed settings in Oculus app (_Im on Quest2_) - but nothing helped.
I don't know how to fix the problem and how to deal with it.

Has anyone encountered and solved it afterwards? It is very unpleasant to play with this bug :(
Thanks.

**Comment by AGluk** [comment]
Hello.

For me, setting SS to 1.0 helped, but the visual quality is so poor :( Almost unplayable.
No one another option affects this problem.

Play on Meta Quest 3 using Oculus mode in MotherVR.

**Comment by Fan4eGGG** [comment]
> For me, setting SS to 1.0 helped, but the visual quality is so poor :( Almost unplayable. No one another option affects this problem.

yeah. I turned down the quality to 1.2 and I'm fine with it, in principle, i beat the game and if you are very soapy picture, I advise you to put in Steam quality higher.

**Comment by JayP236** [comment]
The lighting issues arise from the gameshaders not taking into account the actual resolution of the game. It works fine for resolutions upto 4k, but going above that will cause it to occur.
I created a seperate mod that fixes this issue:
https://github.com/JayP236/Working-Joe-s-Graphics-Improvements-and-Fixes

**Comment by MulleredOG** [comment]
> The lighting issues arise from the gameshaders not taking into account the actual resolution of the game. It works fine for resolutions upto 4k, but going above that will cause it to occur. I created a seperate mod that fixes this issue: https://github.com/JayP236/Working-Joe-s-Graphics-Improvements-and-Fixes

Nice. I'll be checking this out. Just need anti aliasing to work now!

**Comment by JayP236** [comment]
> > The lighting issues arise from the gameshaders not taking into account the actual resolution of the game. It works fine for resolutions upto 4k, but going above that will cause it to occur. I created a seperate mod that fixes this issue: https://github.com/JayP236/Working-Joe-s-Graphics-Improvements-and-Fixes
> 
> Nice. I'll be checking this out. Just need anti aliasing to work now!

I must say that for me playing on VD Godlike preset with pixelscale 1.2, the resolution is high enough so that the missing AA isn't really that annoying anymore. Motion controls is number one on my wishlist ;).

**Comment by MulleredOG** [comment]
Looking forward to trying it. I think I can push 1.3 pixlescale with my RTX 3080. Looking forward to trying your mod over the weekend

**Comment by MulleredOG** [comment]
Works great! Had a few issues at first as I've not played PCVR for a while and it was running like crap. Reinstalled AI, SteamVR and it was still running like crap. Did a clean update of my Nvidia drivers and it ran like a dream, better than I remember it running, was able to get a fairly consistent 120 fps at 1.3 pixel density and 90 fps at 1.4. Settled on 1.3 at 90 as it would still stutter in a few areas. Anti Aliasing is still a bit issue for me, particularly the jaggies around the edges of textures, however the removal of LOD has improved the  experience significantly. The shader rendering is a massive improvement! 

My biggest issue, which was present in the baked in VR mode, and still present with Nibre's mod is the "off" character / head movement. It happens after you crouch for the first time and is present until you restart the game. its particularly noticeable if you crouch near a wall or object and look at your feet and turn. Its like your body has a large turning circle, rather than pivoting on a point. This issue was always there though and wasn't introduced by Nibres mod, so I suspect it cant be fixed easily, similar to the anti aliasing issue.

But great work, you have made the biggest improvements to the VR mode I have seen so far!

**Comment by JayP236** [comment]
Thank you, glad you liked it. I am working on additional VR improvements and fixes, but I won't go into detail about this at the moment. When it becomes more concrete, I'll let the world know :).

**Comment by MulleredOG** [comment]
> Thank you, glad you liked it. I am working on additional VR improvements and fixes, but I won't go into detail about this at the moment. When it becomes more concrete, I'll let the world know :).

Look forward to seeing what you can achieve. Been playing AI in VR for over 10 years now since the first Oculus development kits were available, and I have to say your improvements are up there with what Nibre achieved with his MotherVR Mod

**Comment by JayP236** [comment]
> > Thank you, glad you liked it. I am working on additional VR improvements and fixes, but I won't go into detail about this at the moment. When it becomes more concrete, I'll let the world know :).
> 
> Look forward to seeing what you can achieve. Been playing AI in VR for over 10 years now since the first Oculus development kits were available, and I have to say your improvements are up there with what Nibre achieved with his MotherVR Mod

More info here: https://discord.com/channels/747967102895390741/1363947657248378880
